src/components/PasswordField/PasswordField.vue

```
@@ -0,0 +1,320 @@
+		 */
+		labelVisible: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * The placeholder of the input. This defaults as the string that's
+		 * passed into the label prop. In order to remove the placeholder,
+		 * pass in an empty string.
+		 */
+		placeholder: {
+			type: String,
+			default: undefined,
+		},
+
+		/**
+		 * Toggles the success state of the component. Adds a checkmark icon.
+		 * this cannot be used together with canClear.
+		 */
+		success: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Toggles the error state of the component. Adds an error icon.
+		 * this cannot be used together with canClear.
+		 */
+		error: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Additional error message
+		 *
+		 * This will be displayed beneath the input field
+		 */
+		helperText: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * The minlength property defines the minimum number of characters
+		 * (as UTF-16 code units) the user can enter
+		 */
+		minlength: {
+			type: Number,
+			default: 0,
+		},
+
+		/**
+		 * The maxlength property defines the maximum number of characters
+		 * (as UTF-16 code units) the user can enter
+		 */
+		maxlength: {
+			type: Number,
+			default: null,
+		},
+
+		/**
+		 * Helps the browser identify the type of password field and to provide
+		 * autocompletion for the password. Allowed values are off, on, new-password
+		 * old-password
+		 *
+		 * By default try to autocomplete with the current password
+		 */
+		autocomplete: {
+			type: String,
+			validator: (value) => [
+				'new-password',
+				'current-password',
+				'one-time-code',
+				'on',
+				'off',
+			].includes(value),
+			default: 'current-password',
+		},
+
+		/**
+		 * Check if the user entered a valid password using the password_policy
+		 * app if available.
+		 *
+		 * Warning: this doesn't replace server side checking and will do nothing
+		 * if the password_policy app is disabled.
+		 */
```

Can we throw an error if this is `true` and the password_policy app is disabled?